File: GXpert/GXpert.Web/Modules/ServerTypes/Content/ContentLanguageForm.ts
import { StringEditor, DateEditor, IntegerEditor, PrefixedContext, initFormType } from "@serenity-is/corelib";

export interface ContentLanguageForm {
    Name: StringEditor;
    Description: StringEditor;
    InsertDate: DateEditor;
    InsertUserId: IntegerEditor;
    UpdateDate: DateEditor;
    UpdateUserId: IntegerEditor;
    IsActive: IntegerEditor;
}

export class ContentLanguageForm extends PrefixedContext {
    static readonly formKey = 'Content.ContentLanguage';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!ContentLanguageForm.init)  {
            ContentLanguageForm.init = true;

            var w0 = StringEditor;
            var w1 = DateEditor;
            var w2 = IntegerEditor;

            initFormType(ContentLanguageForm, [
                'Name', w0,
                'Description', w0,
                'InsertDate', w1,
                'InsertUserId', w2,
                'UpdateDate', w1,
                'UpdateUserId', w2,
                'IsActive', w2
            ]);
        }
    }
}

File: GXpert/GXpert.Web/Modules/Content/ContentLanguage/ContentLanguageForm.cs
using Serenity.ComponentModel;
using System;

namespace GXpert.Content.Forms;

[FormScript("Content.ContentLanguage")]
[BasedOnRow(typeof(ContentLanguageRow), CheckNames = true)]
public class ContentLanguageForm
{
    public string Name { get; set; }
    public string Description { get; set; }
    public DateTime InsertDate { get; set; }
    public int InsertUserId { get; set; }
    public DateTime UpdateDate { get; set; }
    public int UpdateUserId { get; set; }
    public short IsActive { get; set; }
}